flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/runtime/batch/sql/MatchRecognizeITCase.java

```diff
@@ -567,7 +912,7 @@ void testUserDefinedFunctions() {
                                 "SELECT *\n"
                                         + "FROM MyTable\n"
                                         + "MATCH_RECOGNIZE (\n"
-                                        + "  ORDER BY proctime\n"
+                                        + "  ORDER BY ts\n"
```

Comment:
As a question, do we need to change all of the tests from using PROCTIME() to ts?

E.g., can the existing behavior remain?

Reply:
> As a question, do we need to change all of the tests from using PROCTIME() to ts?
> 
> E.g., can the existing behavior remain?

If you ask me, MATCH_RECOGNIZE using proctime() in batch mode does not make any sense so I think there is no need to have so many tests for it. But maybe I don't understand the big picture.

If you wish, I can revert my changes and have similar tests for both proctime and event-time.

Reply:
@dawidwys should probably have the final say on that.